ipctest/src/main.rs

```diff
@@ -27,14 +27,34 @@ use crate::errors::*;
 // Run with 'RUST_LOG=run,audioipc cargo run -p ipctest'
 #[cfg(unix)]
 fn run() -> Result<()> {
+    use std::ffi::CString;
+
     let handle =
         unsafe { audioipc_server::audioipc_server_start(std::ptr::null(), std::ptr::null()) };
     let fd = audioipc_server::audioipc_server_new_client(handle);
+    let fd = unsafe {
+        let new_fd = libc::dup(fd);
+        libc::close(fd);
+        new_fd
+    };
+    assert!(fd >= 0);
```

Comment:
`fd > INVALID_HANDLE_VALUE` might work since `INVALID_HANDLE_VALUE` is a `pub const` now?